biopandas/pdb/tests/test_read_pdb.py

```diff
@@ -173,6 +175,19 @@ def test_fetch_af2():
         )
 
 
+def test_read_pdb_esmfold():
+    """Test retrieving a structure from ESMFold."""
+    sequence = "MTYGLY"
+    res_ids: Set[str] = {"A:MET:1", "A:THR:2", "A:TYR:3", "A:GLY:4", "A:LEU:5", "A:TYR:6"}
+    ppdb = PandasPdb().fetch_pdb(sequence=sequence)
```

Comment:
Should this include ESMfold as the `source`? Just as a contingency for when new versions of ESMfold come out. Or if alternatives come up. (But I suppose right now fetching an AlphaFold structure via sequence is not possible via AlphaFold, right?)

Reply:
Yeah, no API for AF2 inference AFAIK

There is a `version` arg for the ESMFold function but it remains to see exactly how the API will change for subsequent releases so may be okay to leave for the time being.